Automattic/newspack-theme
# Style 3: Move the all-caps styles to the customizer option.

### All Submissions:


### Changes proposed in this Pull Request:

A continuation of the work started in #303, this PR moves the text-transform styles in Style 3, so they can optionally be turned off in the Customizer, instead of always applied.

See #280.

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`.
2. Navigate to Customizer > Typography and uncheck "Use all-caps for accent text"
3. Check the following locations and pages, to confirm that the text is now not forced to uppercase:

**Article block section title** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63643109-d5ffc980-c67e-11e9-9646-9ffea91ad835.png)

**Article block section title** (box checked)

![image](https://user-images.githubusercontent.com/177561/63643106-c54f5380-c67e-11e9-9473-b32e36e95a13.png)

**Single Post - category** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63643119-fa5ba600-c67e-11e9-9347-79768d3710b3.png)

**Single Post - category** (box checked)

![image](https://user-images.githubusercontent.com/177561/63643114-e1eb8b80-c67e-11e9-811a-be589b4bee3d.png)

**Single Post - byline & date** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63643120-fdef2d00-c67e-11e9-81a9-805891d3ff48.png)

**Single Post - byline & date** (box checked)

![image](https://user-images.githubusercontent.com/177561/63643113-e0ba5e80-c67e-11e9-9a73-65dbcba723c3.png)

**Single Post - author bio** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63643123-07789500-c67f-11e9-9e16-15a5a26a2a93.png)

**Single Post - author bio** (box checked)

![image](https://user-images.githubusercontent.com/177561/63643129-1bbc9200-c67f-11e9-9aae-10d9e945a099.png)

**Single Post - tag title** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63643158-6a6a2c00-c67f-11e9-8b9e-b8c70c3764f7.png)

**Single Post - tag title** (box checked)

![image](https://user-images.githubusercontent.com/177561/63643137-27a85400-c67f-11e9-983f-0dcc9ade1b50.png)

**Block - Pullquote Citation** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63643153-5b837980-c67f-11e9-95ab-e8d6318af008.png)

**Block - Pullquote Citation** (box checked)

![image](https://user-images.githubusercontent.com/177561/63643138-3131bc00-c67f-11e9-817e-55ce7d45148d.png)

**Footer Widget title** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63643149-4dcdf400-c67f-11e9-9371-59eb5cea46c3.png)

**Footer Widget title** (box checked)

![image](https://user-images.githubusercontent.com/177561/63643142-368f0680-c67f-11e9-99b3-0c1659cb077a.png)

**Footer - site info** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63643151-545c6b80-c67f-11e9-9ed2-d4f36f53d346.png)

**Footer - site info** (box checked)

![image](https://user-images.githubusercontent.com/177561/63643145-3c84e780-c67f-11e9-9a32-f3be2ddf43d9.png)

### Other information:
